doc/tags/deprecated.rst

```diff
@@ -0,0 +1,30 @@
+``deprecated``
+==============
+
+.. versionadded:: 1.x
+    The ``deprecated`` tag was added in Twig 1.x.
```

Comment:
It should say `1.36 and 2.5`. `Added in 1.x` is worthless (and does not mention when it was added in the 1.x branch)

Reply:
This was a draft, thanks.